### ISSUE TITLE
Add logging

### ISSUE THREAD

**Issue by olli-shamlin** [olli-shamlin/flosetta_v1#3]
Implement some consistent logging using [Python's logging facility](https://docs.python.org/3/library/logging.html).

**Comment by olli-shamlin** [comment]
This reference is proving helpful in attempting to implement a function/method tracing method:

-  https://adityadubey.cloud/stack-frames-in-python-accessing-the-frame-object-using-decorator

**Comment by olli-shamlin** [comment]
I've spent enough time for now on implementing a tracer decorator.

At present, the tracer code I have is working reasonably well with at least "unbound" functions that don't have other decorators.  I may be seeing some inaccuracies in the trace message contents; ie, some module/class names seem suspicious. And applying the tracer decorator to functions with other decorators seems to be problematic; eg, I tried applying it to the route functions yesterday and that caused errors. Note I made one slight change to the tracer code this AM that may fix whatever was wrong.  Regardless, for now I have that "unbound" function defined in dbms/\_\_init__.py decorated and that seems to be working.  I pushed the current related code above in the commit above ([6ab7c05](https://github.com/olli-shamlin/flosetta/commit/6ab7c05244f118d916825134cd1d6a1efa6af642)).
